engine/src/main/java/io/camunda/zeebe/engine/processing/resource/ResourceDeletionProcessor.java

```diff
@@ -177,6 +211,45 @@ private void deleteProcess(final DeployedProcess process) {
     }
   }
 
+  private void unsubscribeStartEvents(final DeployedProcess deployedProcess) {
```

Comment:
```suggestion
  private void unsubscribeTimerStartEvents(final DeployedProcess deployedProcess) {
```

Reply:
❓ What do you think about this name ?

Reply:
I get why you say this, but I named it this way because we will unsubscribe different kinds of start event when we implement #9798 and #13354, which I will add to this method. So this name is future-proof 😄